### PR TITLE
[视客营任务] Add ci build results to READEME

### PR DESCRIPTION
**Pull request by xin-li-67**
## Motivation

视客营任务: Add ci build results to READEME.md

## Modification

README.md
README_zh-CN.md
docs/en/01-how-to-build/build-from-source.md
docs/zh_cn/01-how-to-build/build-from-source.md

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by xin-li-67** [comment]
Issues so far:

1. failed to find build status icons for `backend_riscv64_gcc` and `backend_rknpu`
2. I don't know where to add these icons
3. In the `Action` page, both `backend_ort` and `backend_torchscript` use the same tag name as `backend_ort`, even though one is `backend_ort.yml` and the other is `backend_torchscript.yml`

File: README.md [hunk 1]
```diff
@@ -70,6 +70,23 @@ The benchmark can be found from [here](docs/en/03-benchmark/benchmark.md)
 | Adreno GPU        | -                                                               | -                                       | -        | ✔️ncnn<br>✔️SNPE |
 | Hexagon DSP       | -                                                               | -                                       | -        | ✔️SNPE           |
 
+[![Build Status][pass-backend-ort]][ci-backend-ort]
```
Comment (lvhan028) [inline README.md:73]: May put the build results in the above matrix

Comment (xin-li-67) [reply README.md:86]: > May put the build results in the above matrix

Please take a look at the `README.md`. I tried putting the icons there, but it's kind of ugly. LOL

Comment (lvhan028) [reply README.md:86]: Try to put the build badge at the beginning

Comment (xin-li-67) [reply README.md:86]: > Try to put the build badge at the beginning

Moved the badges to the front of each framework

Comment (lvhan028) [reply README.md:86]: Besides `passing` and `failing`, is there another status like `TODO`?

Comment (xin-li-67) [reply README.md:86]: > Besides `passing` and `failing`, is there another status like `TODO`?

These build status icons provided by img.shileds.io need to be specified by an exact build action yml URL. Therefore, there are only four states including `passing`, `falling`, `no status`, and cannot find such an action. I tried to put a `no status` that links to rknpu2-gcc in front of those without action. Not ideal and may cause misleading to others.

Comment (lvhan028) [reply README.md:86]: Let's try to remove "✔️"

File: README.md [hunk 2]
```diff
@@ -68,7 +68,31 @@ The benchmark can be found from [here](docs/en/03-benchmark/benchmark.md)
...
+| Hexagon DSP       | -                                                                                                                                                                                                                                                                                                                                                   | -                                                                                                                                                                                               | -                                                                 | [![Build Status][pass-backend-snpe]][ci-backend-snpe]✔️SNPE                                                              |
+
+[![Build Status][pass-backend-ort]][ci-backend-ort]
+[![Build Status][pass-backend-pplnn]][ci-backend-pplnn]
```
Comment (lvhan028) [inline README.md:87]: Are these lines 87~95 necessary?

Comment (xin-li-67) [reply README.md]: > Are these lines 87~95 necessary?

no they're not

**Comment by lvhan028** [comment]
Could you open another PR for the master branch?

**Pull request closed without merging**